src/cmd/mod.rs

```diff
@@ -148,8 +178,24 @@ pub fn load_wallet(
     }
 
     let config = load_config(wallet_dir).context("loading configuration")?;
-    let keychain = match config.keys {
-        crate::config::WalletKeys::SeedWordsFile => {
+    let database = {
+        let mut db_file = wallet_dir.to_path_buf();
+        db_file.push("database.sled");
+        sled::open(db_file.to_str().unwrap()).context("opening database.sled")?
+    };
+
+    let wallet_db = database
+        .open_tree("wallet")
+        .context("opening wallet tree")?;
+
+    let esplora = match AnyBlockchain::from_config(&config.blockchain)? {
+        AnyBlockchain::Esplora(esplora) => esplora,
+        #[allow(unreachable_patterns)]
+        _ => return Err(anyhow!("A the moment only esplora is supported")),
```

Comment:
```suggestion
        _ => return Err(anyhow!("At the moment only esplora is supported")),
```

Not your typo I know!